showcase/treemap/tree-tools.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Find branch in tree by omen pair - key-value
+ * @author Akiyamka <akiyamka@gmail.com>
```

Comment:
don't include yourself here, this file should have the same open source liscence at the top as all of the other file. copy and paste it from somewhere